_posts/en/newsletters/2020-01-29-newsletter.md

```diff
@@ -0,0 +1,215 @@
+  Teinturier [posted][teinturier post] to the Lightning-Dev mailing list about changing
+  what data is added to a [BOLT11][] invoice for payments sent to
+  unannounced channels---channels that aren't advertised to the LN
+  network and which usually don't route payments for other users.  The
+  proposed change would remove information from an invoice that could be
+  used to identify the channel's deposit UTXO, replacing it with a
+  one-time per-invoice keypair and a secret that's routed as part of the
+  onion-encrypted payment.  This would require special support from both the
+  spender and the peer who's able to route to the unannounced channel,
+  but other nodes along the routing path would not need to change their
+  implementation.  Teinturier requested feedback on the proposal,
+  including any suggestions for how to eliminate the need to include an
+  encrypted secret in the payment.
+
+- **Layered commitments with eltoo:** Anthony Towns [described][towns
+  layered commitments] a potential modification to his previous
+  [anyprevout proposal][bip-anyprevout] (a variation of
+  [SIGHASH_NOINPUT][topic sighash_noinput]) that could simplify
+  [eltoo][topic eltoo]-based LN channels.  As currently proposed,
+  eltoo-based LN implementations would need to ensure they don't accept
+  a payment whose refund timeout occurs before the payment's
+  unilateral-close delay, otherwise a paying node will be able to
+  reclaim its payment before the receiving node has a chance to use
+  their adaptor signature ("point lock") to legitimately claim the
+  payment.  This differs from current-style LN payments where the
+  timeout and the delay may be chosen independently.
+
+    To allow eltoo to achieve a similar independence of timeout and
+    delay parameters, Towns proposes removing the [BIP342][] commitment
+    to the value of the inputs (`sha_amounts`) from signatures created
+    using the `SIGHASH_ANYPREVOUTANYSCRIPT` signature hash (sighash)
+    flag.  This also requires changes to the script used in eltoo,
+    including making use of [tapscript's][topic tapscript] variation of
+    the `OP_CODESEPARATOR` opcode.
+
+## Selected Q&A from Bitcoin StackExchange
+
+*[Bitcoin StackExchange][bitcoin.se] is one of the first places Optech
+contributors look for answers to their questions---or when we have a
+few spare moments to help curious or confused users.  In
+this monthly feature, we highlight some of the top-voted questions and
+answers posted since our last update.*
+
+{% comment %}<!-- https://bitcoin.stackexchange.com/search?tab=votes&q=created%3a1m..%20is%3aanswer -->{%
+endcomment %}
+{% assign bse = "https://bitcoin.stackexchange.com/a/" %}
+
+- [How is OP_CHECKTEMPLATEVERIFY a scaling solution?]({{bse}}92755)
+  Confused_Coder explains ways in which the proposed new opcode, `OP_CHECKTEMPLATEVERIFY`,
+  could help with more [efficient use of block space over time][news48 output commitments]
```

Comment:
I think this bullet is a good summary of the answer using the answer's own terms, but I think it's a bit misleading without all of the context from the answer.  The answer describes how CTV works in the case of empty blocks, but in the case of full blocks, CTV is always less efficient than non-CTV to accomplish the same goal.  Maybe s/more efficient use of block space over time/delaying the use of block space until fees are cheaper/.  Also maybe, s/are then later claimed onchain/can later be expanded to multiple outputs/.

Reply:
Thanks for the suggestions @harding Ive taken them both as I think they are more correct/clear

Reply:
I think the new wording is fine, but I think the old wording was fine too. Congestion control transactions would 'help with more efficient use of the block space over time' in the same way that UTXO consolidation helps - by reducing demand when there's competition for block space and moving that demand to a time when there's less competition. If you accept the premise that block space is not uniformly valuable across time, then arbitraging the use of that block space across time is indeed an efficiency win because block space is taken by users who value it more highly.

I'm skeptical that any exchanges would use the proposed congestion control method, and like I said, new wording is also fine :)